ctgan/data_transformer.py

```diff
@@ -7,9 +7,12 @@
 
 SpanInfo = namedtuple("SpanInfo", ["dim", "activation_fn"])
 ColumnTransformInfo = namedtuple(
-    "ColumnTransformInfo", ["column_name", "column_type",
-                            "transform", "transform_aux",
-                            "output_info", "output_dimensions"])
+    "ColumnTransformInfo", [
+        "column_name", "column_type",
+        "transform", "transform_aux",
+        "output_info", "output_dimensions"
```

Comment:
Bear in mind that this may fail with `flake8-quotes` as we use single quotes now.

Reply:
Yep, I'll fix the conflicts when they show up.